src/staticanalysis/bot/tests/test_remote.py

```diff
@@ -566,6 +566,15 @@ def test_coverity_task(mock_config, mock_revision, mock_workflow):
     assert not issue.is_clang_error()
     assert issue.validates()
 
+    # Testing will coverity full stack support
+    mock_config.cov_full_stack = True
+    issues = mock_workflow.run(mock_revision)
+    issue = issues[0]
+    assert issue.message == 'Using uninitialized value "a".\n' \
```

Comment:
You can use a multiline string variable here to make it nicer to read:

```python
valid_message = '''Using bla
bla
bla
'''
```

Reply:
Sure, I can use raw string literal.